README.md

```diff
@@ -766,9 +766,9 @@ let bananas = L10n.bananasOwner(5, "Olivier")
 
 ---
 
-# License
+# Licence
 
-This code and tool is under the MIT License. See the `LICENSE` file in this repository.
+This code and tool is under the MIT Licence. See the `LICENSE` file in this repository.
```

Comment:
I'm all for consistency between US/UK spelling, but then the LICENSE file should then be renamed. Or we should choose the License spelling. But either way be consistent 😉

Reply:
Yeah I noticed that too, but I'm not sure if GitHub supports it? I'll give it a try and rename it 🤞

Reply:
Mmh no match for `LICENCE` in https://help.github.com/articles/licensing-a-repository/🤔

Reply:
But it is mentioned in https://github.com/benbalter/licensee:
https://github.com/benbalter/licensee/blob/489bc5e1ade015981dffbb1069ec4b9579a96e57/spec/licensee/project_files/license_file_spec.rb#L59